gibbon-compiler/src/Gibbon/Language/Syntax.hs

```diff
@@ -250,7 +312,46 @@ deriving instance (Show (TyOf ex), Show ex, Show (ArrowTy (TyOf ex))) => Show (P
 deriving instance (Eq (TyOf ex), Eq ex, Eq (ArrowTy (TyOf ex))) => Eq (Prog ex)
 deriving instance (Ord (TyOf ex), Ord ex, Ord (ArrowTy (TyOf ex))) => Ord (Prog ex)
 deriving instance Generic (Prog ex)
-deriving instance (NFData (TyOf ex), NFData (ArrowTy (TyOf ex)), NFData ex, Generic (ArrowTy (TyOf ex))) => NFData (Prog ex)
+
+deriving instance
+         (NFData (TyOf ex), NFData (ArrowTy (TyOf ex)), NFData ex,
+          Generic (ArrowTy (TyOf ex))) =>
+         NFData (Prog ex)
```

Comment:
Did you actually edit this line or just reformatted it? If the latter, then better to revert this. If you want to improve formatting, please, submit a separate patch. Alternatively, you could have a separate commit for formatting changes. Currently, there are 73 commits in this PR: that's too many. Please, research (or ask me) how to "squash" commits. One commit is probably enough for the feature (modules) itself. Another commit could be about formatting.

Same applies to all the formatting-only changes, not just this line. E.g. in this file above there are many import-list changes that only reformat it. It better go into a separate commit.